SearchHistory/views.py

```diff
@@ -0,0 +1,8 @@
+from django.shortcuts import render
+# from django.shortcuts import redirect
+# from .models import Message
+# from .forms import MessageForm
```

Comment:
If these lines are not needed, please delete them

Reply:
Done